neqo-transport/src/cc/classic_cc.rs

```diff
@@ -347,8 +347,7 @@ impl<T: WindowAdjustment> CongestionControl for ClassicCongestionControl<T> {
 
     fn discard(&mut self, pkt: &SentPacket) {
         if pkt.cc_outstanding() {
-            assert!(self.bytes_in_flight >= pkt.len());
-            self.bytes_in_flight -= pkt.len();
+            self.bytes_in_flight = self.bytes_in_flight.saturating_sub(pkt.len());
```

Comment:
Under what conditions does this saturate?  Is this only in the case where you PTO?

Reply:
Yes, the extra call to `on_packets_lost` takes packets out of `bytes_in_flight`.